### ISSUE TITLE
test problem

### ISSUE THREAD

**Issue by ljc1231** [LIVIAETS/boundary-loss#30]
early i just use dice loss, when epoch is 87, the val loss is 0.181 and test dice is 0.45 ,then i trained more epochs, the val loss is 0.160, and test dice is only 0.39
the i use your boundary loss, because i use early stopped method , it trained 93 epoches, and the val loss is 0.166, but the test dice is only 0.35 
did the traindataset overfitting ? i use 250 CT and augmented it to 2124 CT.
or the train model is too simple? i used U-net, but it didn't trained well, like the train loss didn't decreaed after only first epoch, i don't konw why?
can you give me some advice, thx!

**Comment by HKervadec** [comment]
Do you have a plot of your loss, training and validation dice over time ? It usually helps a lot to understand the behavior of the model

**Comment by ljc1231** [comment]
i didn't save the above-mentioned experiment loss picture. i only have same data ,use unet ,have below result,but test dice is only 0.3
![image](https://user-images.githubusercontent.com/67781186/102444904-3cbfa800-4065-11eb-91e3-597b4a0a04b9.png)
![image](https://user-images.githubusercontent.com/67781186/102444916-43e6b600-4065-11eb-9a69-81c02054a81f.png)

**Comment by HKervadec** [comment]
I am not sure that I understand. You mention a testing DSC of 0.3, yet the curve you uploaded shows a test DSC of 0.9 ? I find it bizarre as well that the training and testing performances are so close.

**Comment by ljc1231** [comment]
the test dice in picture acutally is the validation dice，i didn’t plot test dice&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by HKervadec** [comment]
Something is off in your training in general, I encourage you to double check everything in your code and how you split the data -- usually there is a bigger gap between training and validation.

You can also take a look visually at the network output, as it might help to diagnose what is going on.
